lib/interfaces/config-module-options.interface.ts

```diff
@@ -1,6 +1,14 @@
 import { ConfigFactory } from './config-factory.interface';
 
 export interface ConfigModuleOptions {
+  /**
+   * If "true" values from process.env are read
+   * only once, the cache will be used after that.
+   * This will improve the performance.
+   * See: https://github.com/nodejs/node/issues/3104
+   */
+  enableCache?: boolean;
```

Comment:
Following the convention from other official NestJS modules, I would suggest using `cache` name instead ("enableCache" is a verb so it rather looks like a function/method)